core/src/components/footer/footer.tsx

```diff
@@ -46,6 +51,30 @@ export class Footer implements ComponentInterface {
     this.checkCollapsibleFooter();
   }
 
+  connectedCallback() {
+    if (typeof (window as any) !== 'undefined') {
+      this.keyboardWillShowHandler = () => {
+        this.keyboardVisible = true;
+      };
+
+      this.keyboardWillHideHandler = () => {
+        setTimeout(() => (this.keyboardVisible = false), 50);
```

Comment:
Is there any way we can avoid a `setTimeout` here? Would request animation frame wrapping this callback work instead?

If not, can we document why the set timeout is used and why the value of `50` was selected? 

Edit: This also looks like maybe copy & paste from the tab bar. Can they share this behavior and also can we investigate what drove a the setTimeout & duration?